infrastructure/cdn-in-a-box/traffic_ops/run-go.sh

```diff
@@ -87,6 +87,12 @@ while true; do
 done
 
 ### Add SSL keys for demo1 delivery service
+until [[ -s "$X509_DEMO1_CERT_FILE" && -s "$X509_DEMO1_REQUEST_FILE" && -s "$X509_DEMO1_KEY_FILE" ]]
+do
+  echo "Waiting on X509_DEMO1 files to exist"
+  sleep 3
```

Comment:
would result in possible infinite loop if the file did not exist for some reason. its better to loop for certain duration or until the file exists.

Reply:
That's true, but because the processes are totally asynchronous there's know way to know for an arbitrary host what a reasonable timeout would be.

Reply:
Moreover, I think we really do want to loop forever, for things required for initialization. If the Docker container can't proceed without this, looping forever and continuously logging what's being waited on makes it easier to debug what went wrong.

Reply:
Yeah, the general async architecture of CIAB tends to require waiting on things indefinitely. That can be frustrating sometimes though, because it's not always clear how long CIAB is "supposed to take" in order to fully startup, and it's not immediately obvious when CIAB has actually started up properly from the logs.

That said, this PR jives w/ the existing architecture, and if we to change the architecture, we should discuss/design that in a more open forum.